specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3711,20 +3666,25 @@
           "description": "The address associated with billing account.",
           "$ref": "#/definitions/Address"
         },
+        "company": {
+          "description": "Company Name.",
+          "type": "string",
+          "readOnly": true
+        },
         "country": {
-          "description": "The country associated with billing account..",
+          "description": "Country Name.",
```

Comment:
egIn the examples below, it looks like the country code (eg "US") is used instead of the country name (eg "United States"). Should this be changed to "Country code"?

Reply:
The country field has the full country name in Api response.
Please refer the example: BillingAccountsListWithExpandForEnrollmentDetails.json
"country": "United States"